connect/runtime/src/main/java/org/apache/kafka/connect/runtime/distributed/WorkerGroupMember.java

```diff
@@ -203,8 +203,8 @@ public void requestRejoin() {
         coordinator.requestRejoin();
     }
 
-    public void maybeLeaveGroup(String leaveReason) {
-        coordinator.maybeLeaveGroup(leaveReason);
+    public void maybeLeaveGroup(String leaveReason, boolean shouldWarn) {
```

Comment:
It seems the `shouldWarn` is always `true` in production. Maybe we can remove the input argument from this method?

Reply:
Nice suggestion!